src/components/content-tab-import-export.tsx

```diff
@@ -305,18 +304,11 @@ const ImportSite = ( {
 
 export function ContentTabImportExport( { selectedSite }: ContentTabImportExportProps ) {
 	const [ isSupported, setIsSupported ] = useState< boolean | null >( null );
-	const { isAnySitePulling, isAnySitePushing, getPullState, getPushState, connectedSites } =
+	const { isAnySitePulling, isAnySitePushing, isSiteIdPulling, isSiteIdPushing, connectedSites } =
 		useSyncSites();
-	const { isKeyPulling } = useSyncStatesProgressInfo();
 	const isAnySiteSyncing = isAnySitePulling || isAnySitePushing;
-	const isPulling = connectedSites.some( ( site ) => {
-		const sitePullState = getPullState( selectedSite.id, site.id );
-		return sitePullState && isKeyPulling( sitePullState.status.key );
-	} );
-	const isPushing = connectedSites.some( ( site ) => {
-		const sitePushState = getPushState( selectedSite.id, site.id );
-		return sitePushState?.isInProgress;
-	} );
+	const isPulling = connectedSites.some( ( site ) => isSiteIdPulling( selectedSite.id, site.id ) );
+	const isPushing = connectedSites.some( ( site ) => isSiteIdPushing( selectedSite.id, site.id ) );
 	const isThisSiteSyncing = isPulling || isPushing;
```

Comment:
I see that we are using this logic in a couple of different files. I am wondering if we might want to create a hook to be able to reuse this check. This is not directly related to this PR though as this was already present before. We can create a separate issue to consider this.